specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc

```diff
@@ -1177,11 +1173,11 @@ automatically choose a unique name or allow the Deployer to choose a
 unique name, but must not fail the deployment. This ensures that
 existing modules continue to be deployable.
 . The deployment tool must deploy all of the
-components listed in the Java EE module deployment descriptor, or marked
+components listed in the Jakarta EE module deployment descriptor, or marked
 via annotations and discovered as described in the previous requirement,
-according to the deployment requirements of the respective Java EE
+according to the deployment requirements of the respective Jakarta EE
 component specification. If the module is a type that contains JAR
-format files (for example, web and Connector modules), all classes in
+format files (for example, web and Jakarta Connector modules), all classes in
```

Comment:
This was just a generic reference, I would have just lowercased "connector".

Reply:
done